battery.cpp

```diff
@@ -7,6 +7,9 @@ void Battery::setup(Ui::MainWindow *ui) {
   this->ui->battery->setValue(battery);
   range = battery*efficiencyConstant;
   ui->range->display(int(range));
+
+  lowBatteryIcon = QPixmap(":/icons/low-battery.png");
```

Comment:
Make sure to add the `low-battery.png` into the `icons.qrc` file in order for it to work as a QT resource